.github/workflows/ci.yml

```diff
@@ -112,7 +112,7 @@ jobs:
           name: opentelemetry-go-test-output
           path: ${{ env.TEST_RESULTS }}
 
-  compatibility-test:
+  compatibility-matrix:
```

Comment:
Changing this name means the existing required check will not be run. The original name should be kept here and if we want to change it we can do so once the PR required checks are moved to the require test-compatibility.

Reply:
oh good catch, I reverted it back to the original name.